Library/Formula/llvm.rb

```diff
@@ -114,12 +114,12 @@ def install
       ENV["UNIVERSAL_ARCH"] = Hardware::CPU.universal_archs.join(" ")
     end
 
-    ENV["REQUIRES_RTTI"] = "1" if build.with?("rtti") || build.with?("clang")
-
     args = %w[
       -DLLVM_OPTIMIZED_TABLEGEN=On
     ]
 
+    args << "-DLLVM_ENABLE_RTTI=True" if build.with?("rtti") || build.with?("clang")
```

Comment:
I guess you are right, although the [llvm cmake docu](http://llvm.org/docs/CMake.html) with ```LLVM_ENABLE_RTTI:BOOL``` is inaccurate.